packages/mask/src/plugins/CryptoartAI/apis/index.ts

```diff
@@ -0,0 +1,113 @@
+import { OrderSide } from 'opensea-js/lib/types'
+import type { ChainId } from '@masknet/web3-shared-evm'
+import urlcat from 'urlcat'
+import { resolveAPILinkOnCryptoartAI } from '../pipes'
+
+export async function getAsset(tokenId: string, chainId?: ChainId) {
+    const ownersResponse: any = await (
+        await fetch(
+            urlcat(resolveAPILinkOnCryptoartAI(chainId), '/api/artwork/current/owners/:tokenId', { tokenId: tokenId }),
+            {
+                mode: 'cors',
+                headers: {
+                    'Content-Type': 'application/json; charset=UTF-8',
+                    Accept: 'application/json, text/plain, */*',
+                },
+            },
+        )
+    ).json()
+
+    const tradeResponse: any = await (
+        await fetch(
+            urlcat(resolveAPILinkOnCryptoartAI(chainId), '/api/artwork/tradeInfo/:tokenId', { tokenId: tokenId }),
+            {
+                mode: 'cors',
+                headers: {
+                    'Content-Type': 'application/json; charset=UTF-8',
+                    Accept: 'application/json, text/plain, */*',
+                },
+            },
+        )
+    ).json()
+
+    const fetchResponse = await (
+        await fetch(urlcat(resolveAPILinkOnCryptoartAI(chainId), '/api/artwork/detail'), {
+            mode: 'cors',
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json; charset=UTF-8',
+                Accept: 'application/json, text/plain, */*',
+            },
+            body: JSON.stringify({
+                artworkId: tokenId,
+                ethAddress: '',
+            }),
+        })
+    ).json()
+
+    return {
+        ...fetchResponse.data,
+        owners:
+            ownersResponse.data && ownersResponse.data.length === 0 && fetchResponse.data.creatorInfo
+                ? [
+                      {
+                          ownerAddress: fetchResponse.data.creatorInfo.ethAddress,
+                          ownerName: fetchResponse.data.creatorInfo.username,
+                          ownerAvator: fetchResponse.data.creatorInfo.avatorPath,
+                          ownerRoleType: fetchResponse.data.creatorInfo.roleType,
+                      },
+                  ]
+                : ownersResponse.data,
+        trade: tradeResponse.data,
+    }
+}
+
+export async function getEvents(tokenId: string, chainId?: ChainId) {
+    const historyResponse: any = await (
+        await fetch(
+            urlcat(resolveAPILinkOnCryptoartAI(chainId), '/api/artwork/recent/history/:tokenId', { tokenId: tokenId }),
```

Comment:
```suggestion
            urlcat(resolveAPILinkOnCryptoartAI(chainId), '/api/artwork/recent/history/:tokenId', { tokenId }),
```

see http://es6-features.org/#PropertyShorthand